src/parse-options.js

```diff
@@ -1,18 +1,33 @@
 import {resolve} from 'path'
+import styles from 'chalk'
 
 const DEFAULT_OPTIONS = {
   nodeBin: 'node',
   maxAttempts: 3,
   protractorArgs: [],
+  // set color to true to use a default color or set to one of the colors available at 'chalk' - https://github.com/chalk/ansi-styles#colors
+  color: null,
```

Comment:
Actually, maybe we just make this `magenta` and then we can eliminate the need to check for `true`; we just ensure that color is not false and is part of styles